### PR TITLE
Added UCB's 61A 

### PR DESCRIPTION
**Pull request by tnb24**
## What does this PR do?
Add 61A as recorded in 2011 in UC Berkeley

### Description

Added 61A as recorded in 2011

### Why is this valuable (or not)?

It's valuable as it explains SICP (which is also a great free book) roughly chapter by chapter

### How do we know it's really free?
Nothing is paywalled on the archive page

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by tnb24** [comment]
Don't see why is it failing

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -560,6 +560,7 @@
 * [AI Courses](https://software.intel.com/content/www/us/en/develop/topics/ai/training/courses.html) - Intel Corporation
 * [Algorithm Design and Implementation](https://www.youtube.com/playlist?list=PL6EF0274BD849A7D5)
 * [Aml-2018 Ambient Intelligence](https://www.youtube.com/playlist?list=PLqRTLlwsxDL8fUcY2Y54sITILyJcTySpC) (F. Corno - L. De Russis - A. Monge Roffarello)
+* [Berkeley's CS 61A: Taught using SICP](https://archive.org/details/ucberkeley-webcast-PL3E89002AA9B9879E?tab=collection)
 * [Berkeley's CS  61B: Data Structures](https://archive.org/details/ucberkeley_webcast_QMV45tHCYNI)
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:564]:
```suggestion
* [Berkeley's CS 61B: Data Structures](https://archive.org/details/ucberkeley_webcast_QMV45tHCYNI)
```

Maybe the double space between "CS" and "61B" could cause problems, try to remove one space.

**Comment by tnb24** [comment]
Fixed all errors. It works

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**